<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSIRO::OpenCV::MatWidget</class>
 <widget class="QWidget" name="CSIRO::OpenCV::MatWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cv::Mat viewer</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="CSIRO::DataAnalysis::ImageWidget" name="imageWidget"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CSIRO::DataAnalysis::ImageWidget</class>
   <extends>QFrame</extends>
   <header>DataAnalysis/Image/imagewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
